python/ray/tests/test_logging_2.py

```diff
@@ -221,6 +239,7 @@ def print_message(self):
 """
         stderr = run_string_as_driver(script)
         should_exist = [
+            "timestamp_ns",
             "job_id",
             "worker_id",
             "node_id",
```

Comment:
Non-blocker: would be nice to freeze time and check the actual value of the `timestamp_ns` in the log. And also to ensure it's an int 😄

Reply:
update https://github.com/ray-project/ray/pull/46074/commits/2504ab802c5ee0bfaac1f24b35317e36d32fa995